IBAnimatable/MaskDesignable.swift

```diff
@@ -50,7 +51,9 @@ public extension MaskDesignable where Self: UIView {
     case .triangle:
       maskTriangle()
     case .none:
-      layer.mask?.removeFromSuperlayer()
+      if previousMaskType != nil {
+        layer.mask?.removeFromSuperlayer()
```

Comment:
Another alternative (I believe) would be to have `previousMaskType : MaskType` initialized to `.none` and:
```
      if case .none = previousMaskType {
        layer.mask?.removeFromSuperlayer()
      }
```